### PR TITLE
Update VRBangers.yml

### PR DESCRIPTION
**Pull request by Foucoubou**
Fix studio scrape name - (using replace) in order to have name same as in stashdb
Fix performers nort scraping

### CODE REVIEW

**Comment by Foucoubou** [comment]
@Belleyy  hopefully done :)

File: scrapers/VRBangers.yml [hunk 1]
```diff
@@ -25,16 +26,18 @@ xPathScrapers:
...
+        Name: //div[starts-with(@class, 'video-item__info-starring')]//a/text()
       Studio:
         Name: &studioName
           selector: //meta[@name="dl8-customization-brand-name"]/@content
           postProcess:
             - replace:
                 - regex: \#
                   with: ""
+            - map:
+                VR Bangers: "VRBangers"
```
Comment (halorrr) [inline scrapers/VRBangers.yml:40]: StashDB is wrong here, I'll submit an edit, this should have a space between the words.

As a side note, I think whether or not a studio has a space between words is often more of the job for a studio alias in Stash than the scraper itself.

Comment (halorrr) [reply scrapers/VRBangers.yml:40]: Edit to fix the StashDB side here https://stashdb.org/edits/21b90e1f-dc7b-4500-8cea-11838411af76

Comment (Foucoubou) [reply scrapers/VRBangers.yml]: I have removed the map then :)
Thanks for your insight.

File: scrapers/VRBangers.yml [hunk 2]
```diff
@@ -25,16 +26,18 @@ xPathScrapers:
         selector: //div[contains(@class,"second-text")]/div//text()
         concat: " "
       Tags:
-        Name: //div[starts-with(@class,"video-item__tags ")]//a/text()
+        Name: //a[contains(@href,"category/")]/text()
```
Comment (bnkai) [inline scrapers/VRBangers.yml:29]: This need to be more specific.
In all of the sites category anchors are sometimes used in the description with different text that shouldnt be matched
eg
`https://vrconk.com/video/pocahontas-a-xxx-parody/` , `https://vrbangers.com/video/yoga-hosers/` , `https://vrbgay.com/video/desperate-housedudes/` , `https://vrbtrans.com/video/the-best-cumshot-compilation/`.
Notice the anchors in the description and the tags you get when scraping...
The following seems to work (limited testing though )
```suggestion
        Name: //div[@data-testid="video-categories-list"]/a[contains(@href,"category/")]/text()
```

File: scrapers/VRBangers.yml (hunk 1 above)
Comment (bnkai) [reply scrapers/VRBangers.yml]: Mapping should be removed as @halorrr  suggests

**Comment by Foucoubou** [comment]
It is strange I have tested this and it was working....

**Pull request merged**